docs/usage/management_commands.rst

```diff
@@ -37,4 +80,15 @@ How to use
 
 .. code-block:: bash
 
-  ./manage.py migrate_with_timeouts --lock-timeout-in-ms=10000 --statement-timeout-in-ms=60000
+  ./manage.py migrate_with_timeouts \
+    --lock-timeout-in-ms=10000 \
+    --lock-timeout-max-retries=3 \
+    --lock-timeout-retry-exp=2 \
+    --lock-timeout-retry-min-wait-in-ms=3000 \
+    --lock-timeout-retry-max-wait-in-ms=10000 \
+    --statement-timeout-in-ms=10000 \
+    --statement-timeout-max-retries=3 \
+    --statement-timeout-retry-exp=2 \
+    --statement-timeout-retry-min-wait-in-ms=3000 \
+    --statement-timeout-retry-max-wait-in-ms=10000 \
+    --retry-callback--path="dotted.path.to.callback.function"
```

Comment:
♻️

Reply:
Refer to https://github.com/kraken-tech/django-pg-migration-tools/pull/20#discussion_r1760473678